<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputCustomParam</class>
 <widget class="QDialog" name="InputCustomParam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="custom_params_box">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>10</y>
     <width>411</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="custom_params_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Use custom parameters</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="a_input">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_input">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="r0_input">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="default_params_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Use default parameters</string>
    </property>
   </widget>
   <widget class="QLabel" name="a_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>m</string>
    </property>
   </widget>
   <widget class="QLabel" name="r0_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>r0</string>
    </property>
   </widget>
   <widget class="QLabel" name="n_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>n</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="n_input">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="submit_button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
